com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/validations/SourceInfoLevels.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.ca.lsp.cobol.service.delegates.validations;
+
+import lombok.AllArgsConstructor;
+
+/**
+ * This class contains human-readable text describing the source of diagnostics, including severity
+ * level.
+ */
+@AllArgsConstructor
+public enum SourceInfoLevels {
+  ERROR("E"),
+  WARNING("W"),
+  INFO("I"),
+  HINT("I");
```

Comment:
"I", I guess point is "Hint" is "Info" at the end, right?

Reply:
Oh, it's a typo, my bad. 
The order is Error > Warning > Info > Hint